translations/credits.tex/es.po

```diff
@@ -32,6 +32,8 @@ msgid ""
 "{2}\n"
 "\n"
 msgstr ""
+"{2}\n"
+"\n"
```

Comment:
```suggestion
```
Don't translate those without any text to translate. `po4a` will pick up the original automatically.

Reply:
The text: MANUAL DE REGLAS REESCRITO

Reply:
Pushed!

Reply:
Please delete those translations with no actual text.